### PR TITLE
Edx abolger/ent 3559 learner signals

### PR DESCRIPTION
**Pull request by edx-abolger**
**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/edx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: tests/test_utilities.py [hunk 1]
```diff
@@ -827,6 +828,55 @@ def test_get_enterprise_customer_for_user(self):
...
+            enterprise_customer_user__user_id=user.id,
+            course_id=course_run_id,
+            enterprise_customer_user__enterprise_customer__active=False
+        )
+        expected_uuid4 = str(ece4.enterprise_customer_user.enterprise_customer.uuid)
+        self.assertNotIn(expected_uuid4, utils.get_enterprise_uuids_for_user_and_course(user, course_run_id, active=True))
+        self.assertIn(expected_uuid4, utils.get_enterprise_uuids_for_user_and_course(user, course_run_id))
+        self.assertIn(expected_uuid4, utils.get_enterprise_uuids_for_user_and_course(user, course_run_id, active=False))
+
+        # Needed for user?
```
Comment (alex-sheehan-edx) [inline tests/test_utilities.py:874]: unsure about this comment, what is needed for the enterprise user?

**Review by alex-sheehan-edx — APPROVED** [review]
One question on a commented out section but otherwise looks good to me 👍

File: integrated_channels/integrated_channel/tasks.py [hunk 2]
```diff
@@ -149,28 +157,32 @@ def transmit_single_subsection_learner_data(username, course_run_id, subsection_
...
-    duration = time.time() - start
-    LOGGER.info(
-        '[Integrated Channel] Single learner data transmission task finished.'
-        ' Course: {course_run}, Duration: {duration}, Username: {username}'.format(
-            username=username,
-            course_run=course_run_id,
-            duration=duration))
+    # Transmit the learner data to each integrated channel for each related customer.
+    # Starting Export. N customer is usually 1 but multiple are supported in codebase.
+    for enterprise_customer_uuid in enterprise_customer_uuids:
```
Comment (alex-sheehan-edx) [inline integrated_channels/integrated_channel/tasks.py:165]: is this supposed to be `enterprise_uuids`?

File: tests/test_utilities.py (hunk 1 above)
Comment (edx-abolger) [reply tests/test_utilities.py:874]: Good catch, the comment was for the commented out code below it which needs removing.

File: integrated_channels/integrated_channel/tasks.py (hunk 2 above)
Comment (edx-abolger) [reply integrated_channels/integrated_channel/tasks.py:165]: So... the data model is EnterpriseCustomer. Buuut... pylint complained it was too long when I named the function get_enterprise_customer_uuids_for_user_and_course.  Suggestions welcome?

**Comment by edx-abolger** [comment]
Note: when I attempted to run the checklist, numerous CSS changes and point releases were pulled in as part of the upgrade that have nothing to do with my PR. As a result I created new maintenance PRs for those items and my PR will be merged without the.

**Pull request merged**